runtime/ts/storage/storage-provider-base.ts

```diff
@@ -147,7 +148,7 @@ export abstract class StorageProviderBase {
     return 0;
   }
 
-  toString(handleTags): string {
+  toString(handleTags: string[]): string {
```

Comment:
if handleTags is required to be an array already, probably don't need to re-spread it below.

Reply:
done